usr/share/rear/restore/DUPLICITY/default/400_restore_duplicity.sh

```diff
@@ -43,11 +43,11 @@ if [ "$BACKUP_PROG" = "duplicity" ]; then
 	
     LogPrint "Logging to $TMP_DIR/duplicity-restore.log"
     if [[ -n "${BACKUP_DUPLICITY_GPG_OPTIONS}" ]] ; then
-        LogPrint "with CMD: $DUPLICITY_PROG -v 5 $GPG_KEY --gpg-options ${BACKUP_DUPLICITY_GPG_OPTIONS} --force --tempdir=$DUPLICITY_TEMPDIR $BACKUP_DUPLICITY_URL/$HOSTNAME/ $TARGET_FS_ROOT"
-        $DUPLICITY_PROG -v 5 $GPG_KEY --gpg-options "${BACKUP_DUPLICITY_GPG_OPTIONS}" --force --tempdir="$DUPLICITY_TEMPDIR" $BACKUP_DUPLICITY_URL/$HOSTNAME/ $TARGET_FS_ROOT 0<&6 | tee $TMP_DIR/duplicity-restore.log
+        LogPrint "with CMD: $DUPLICITY_PROG -v 5 ${BACKUP_DUPLICITY_OPTIONS} $GPG_KEY--gpg-options ${BACKUP_DUPLICITY_GPG_OPTIONS} --force --tempdir=$DUPLICITY_TEMPDIR $BACKUP_DUPLICITY_URL/$HOSTNAME/ $TARGET_FS_ROOT"
```

Comment:
@GreenBlood 
here you changed
```
... $GPG_KEY --gpg-options ...
```
to
```
... $GPG_KEY--gpg-options ...
```
i.e. without the space character
which looks wrong because at other places there is a space after `$GPG_KEY`.
Is that intentional here - and if yes why?

Reply:
Indeed a typo on my part, i've fixed it